src/app/docs/toolbar/toolbar-guidelines.component.html

```diff
@@ -0,0 +1,285 @@
+            {{item.label}}
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{item.value}}
+          </sky-definition-list-value>
+        </sky-definition-list-content>
+      </sky-definition-list>
+    </sky-column>
+  </sky-row>
+
+<sky-docs-design-guideline
+  heading="More actions"
+>
+  <p>
+    Place less important or less-frequently used actions in a more actions menu. And when horizontal space in a toolbar is insufficient for all actions, standard actions should collapse into the more actions menu.
+  </p>
+
+  <div class="sky-emphasized">More button</div>
+  <sky-row>
+    <sky-column screenSmall="3" screenMedium="2">
+      <div style="margin: 5px 0;">
+				<sky-dropdown>
+          <sky-dropdown-button>
+						<i class="fa fa-ellipsis-h" style="margin-right: 5px;"></i> More
+          </sky-dropdown-button>
+					</sky-dropdown>
+			</div>
+    </sky-column>
+    <sky-column screenSmall="9" screenMedium="10">
+      <sky-definition-list>
+        <sky-definition-list-content *ngFor="let item of moreButtonInfo">
+          <sky-definition-list-label>
+            {{item.label}}
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{item.value}}
+          </sky-definition-list-value>
+        </sky-definition-list-content>
+      </sky-definition-list>
+    </sky-column>
+  </sky-row>
+</sky-docs-design-guideline>
+
+<sky-docs-design-guideline
+  heading="Find in this list"
+>
+  <p>
+    When a toolbar includes a search component, it should follow the actions and more actions button. The search field is aligned with the action buttons, so if a user has a really large monitor resolution, the field appears on the left side of the screen.
+  </p>
+
+  <sky-row>
+    <sky-column screenSmall="4" screenMedium="3">
+      <div style="margin: 5px 0;"><input class="sky-form-control" placeholder="Find in this list" style="border-radius: 3px;" /><i class="fa fa-search" style="margin-left: -25px;"></i></div>
+    </sky-column>
+    <sky-column screenSmall="8" screenMedium="9">
+      <sky-definition-list>
+        <sky-definition-list-content *ngFor="let item of findBoxInfo">
+          <sky-definition-list-label>
+            {{item.label}}
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{item.value}}
+          </sky-definition-list-value>
+        </sky-definition-list-content>
+      </sky-definition-list>
+    </sky-column>
+  </sky-row>
+</sky-docs-design-guideline>
+
+<sky-docs-design-guideline
+  heading="Items related to views"
+>
+  <p>
+    The view section is on the right side of the toolbar. It is reserved for items that substantially change or affect the view of the rest of the content container. The items in a view section largely depend on how the toolbar is used. If the view section includes more than one item, they appear from left to right in the following order.
```

Comment:
I'd change the first sentence to `Place the view section on the right side of the toolbar.`

In the second sentence, I'd change `It is reserved for` to `Reserve it for`.

In the third sections, I'd delete `in a view section` because it seems redundant. And what do you think of changing `on how the toolbar is used` to `on the purpose of the toolbar` to avoid passive voice?

I'd change the last sentence to: `View section items appear from left to right in the following order when present in a toolbar:`

Reply:
Following up again. Do you think these changes are worth making, or should we just resolve the conversation?